platform/linux-generic/odp_shared_memory.c

```diff
@@ -17,7 +17,7 @@
 #include <string.h>
 
 /* Supported ODP_SHM_* flags */
-#define SUPPORTED_SHM_FLAGS (ODP_SHM_SW_ONLY | ODP_SHM_PROC | ODP_SHM_SINGLE_VA | ODP_SHM_EXPORT | \
+#define SUPPORTED_SHM_FLAGS (ODP_SHM_PROC | ODP_SHM_SINGLE_VA | ODP_SHM_EXPORT | \
```

Comment:
If an out-of-tree application uses the flag, this change may break it even if deprecated APIs are enabled. Maybe linux-gen should not remove support for a deprecated API without a good reason. Here ODP_SHM_SW_ONLY could be included in the SHM capabilities if ODP is compiled with the deprecated APIs enabled.

Reply:
Fixed in V2.